src/libs/SidebarUtils.js

```diff
@@ -166,7 +177,9 @@ function getOrderedReportIDs(currentReportId, allReportsDict, betas, policies, p
 
     // Now that we have all the reports grouped and sorted, they must be flattened into an array and only return the reportID.
     // The order the arrays are concatenated in matters and will determine the order that the groups are displayed in the sidebar.
-    return _.pluck([].concat(pinnedReports).concat(outstandingIOUReports).concat(draftReports).concat(nonArchivedReports).concat(archivedReports), 'reportID');
+    const LHNReports = [].concat(pinnedReports, outstandingIOUReports, draftReports, nonArchivedReports, archivedReports).map((report) => report.reportID);
+    reportIDsCache.set(cachedReports, LHNReports);
```

Comment:
I'm not seeing when the old keys get removed from the cache, will be good to delete old keys, if not a lot of reports will be in memory and won't be used or accessed anymore.